lib/build.gradle

```diff
@@ -30,33 +29,13 @@ oss {
     }
 }
 
-def javaVersion = 8
-java {
-    toolchain {
-        languageVersion = JavaLanguageVersion.of(11)
-    }
-}
-
 compileJava {
     exclude 'module-info.java'
```

Comment:
this is the only line needed for module support to keep working, otherwise 
![image](https://user-images.githubusercontent.com/3900123/123821739-f55f8180-d8fb-11eb-82d8-106026d06296.png)